daphne/src/roles.rs

```diff
@@ -1028,6 +1039,13 @@ where
                     }
                 };
 
+                self.audit_log().on_aggregation_job_continue_req(
+                    req.host(),
+                    task_id,
+                    task_config,
+                    out_shares_count,
+                );
```

Comment:
Without knowing what we're doing with this information, I don't quite see the value of having two callbacks, one for agg init and one for agg cont. It seems like the same information is being reported.